lib/hanami/cli_base.rb

```diff
@@ -0,0 +1,30 @@
+require 'thor'
+
+module Hanami
+  class CliBase < Thor
```

Comment:
@jodosha I think that base class is more idiomatically for different commands. But also I can agree with you too.